### ISSUE TITLE
non-binary representation in neighbourhood meshes

### ISSUE THREAD

**Issue by allysonryan** [clEsperanto/pyclesperanto_prototype#287]
@haesleinhuepf As we discussed this morning, it might be cool to have the option to draw mesh lines in the image representations of neighbourhoods that can be assigned values reflecting how many times the 'edge' is traversed for networks paths :)

**Comment by haesleinhuepf** [comment]
Yes, awesome! I may need to pick your brain at some point to come up with good names for all these operations.
